docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -124,17 +124,19 @@ follow:
 
 .. code-block:: yaml
 
-  per_filter_config:
+  typed_per_filter_config::
```

Comment:
I guess this second colon is not needed.

Reply:
Yeah, it's a typo.